xml/art_jeos_quickstart.xml

```diff
@@ -216,6 +216,34 @@
     </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>systemd-coredump disabled</term>
+     <listitem>
+      <para>
+       Because of the small amount of memory and disk space available by default on JeOS images,
```

Comment:
```suggestion
       Because of the small amount of memory and disk space available by default on JeOS images,
```

Due to the limited amount of memory and disk space